auth0/src/main/java/com/auth0/android/provider/SignatureVerifier.java

```diff
@@ -41,7 +41,11 @@ void verify(@NonNull JWT token) throws TokenValidationException {
 
     private void checkAlgorithm(String tokenAlgorithm) throws TokenValidationException {
         if (!supportedAlgorithms.contains(tokenAlgorithm)) {
-            throw new TokenValidationException(String.format("Signature algorithm of \"%s\" is not supported. Expected the ID token to be signed with any of %s.", tokenAlgorithm, supportedAlgorithms));
+            if (supportedAlgorithms.size() == 1) {
+                throw new TokenValidationException(String.format("Signature algorithm of \"%s\" is not supported. Expected the ID token to be signed with %s.", tokenAlgorithm, supportedAlgorithms.get(0)));
+            } else {
+                throw new TokenValidationException(String.format("Signature algorithm of \"%s\" is not supported. Expected the ID token to be signed with any of %s.", tokenAlgorithm, supportedAlgorithms));
+            }
```

Comment:
I kept the supported algorithms as an array, in the event additional signing algorithms are supported. I did update the exception message for the usual case, for better readability. That said, I'm open to changing it to a single string if that is preferred.

Reply:
I'm not aware of any plans to support further algorithms. That said, if you rather keep a one-item constant array instead of a constant string, that's OK